main.tex

```diff
@@ -1187,7 +1202,7 @@
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must, at their cost, keep the Lot free from pests, including (without limitation) cockroaches, spiders, ants, rodents and other vermin.
```

Comment:
Why are we required to kill spiders? And ants? How is this even possible? Scatter poison all over the building and kill anything that moves?